src/Bithumb/Spot/API/OrderBook.jl

```diff
@@ -0,0 +1,116 @@
+module OrderBook
+
+export OrderBookQuery,
+    OrderBookData,
+    order_book
+
+using Serde
+using Dates, NanoDates, TimeZones
+
+using CryptoAPIs
+using CryptoAPIs.Bithumb
+using CryptoAPIs.Bithumb: Data
+using CryptoAPIs: Maybe, APIsRequest
```

Comment:
```suggestion
using CryptoAPIs.Bithumb
using CryptoAPIs.Bithumb: Data
using CryptoAPIs: Maybe, APIsRequest
import CryptoAPIs: prepare_json!
```
I suggest we do this for uniformity